lib/slappy/client.rb

```diff
@@ -66,6 +64,16 @@ def schedule(pattern, options = {}, &block)
 
     private
 
+    def setup
+      @start_time = Time.now
```

Comment:
Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.